### DISCUSSION TITLE
How to check if node content is empty?

### DISCUSSION THREAD

**Discussion by yellow1912** [ueberdosis/tiptap#1572] [General Questions] [answered]
Hello team,

I wonder if there is a way to check if the editable content is empty inside the renderHTML function? Right now we only receive the HTMLAttributes in there so there is not way to check if the editable content is empty with that.

For example, lets say that I use the figure extension https://www.tiptap.dev/experiments/figure/, but inside the renderHTML I want to check if the figcaption is empty or not to add some classes.

**Comment by hanspagel** [comment, marked as answer]
https://developer.mozilla.org/en-US/docs/Web/CSS/:empty

**Reply by yellow1912** [reply]
Thank you, I ended up checking node.content.size, (node is actually passed into renderHTML as well.
